testsuite/tests/singlecluster/gateway/reconciliation/change_targetref/conftest.py

```diff
@@ -93,16 +104,13 @@ def dns_policy2(blame, gateway2, module_label, dns_provider_secret, request):
     return policy
 
 
-@pytest.fixture(scope="session")
```

Comment:
Dont remove the fixture. If the scope is too broad you can change it to "module" so it fits. If you want to run the fixture body later in the test you can create a function and return it like it was before.